### ISSUE TITLE
Documentation states that ROOT 5.28 is the ports version (Trac #1362)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1354]
On
http://software.icecube.wisc.edu/offline_trunk/metaproject/quickstart.html
it says that "ROOT (http://root.cern.ch/drupal/) is supplied with the Port-based tool system. Currently we provide ROOT v5.28.00d. The I3Build system will automatically choose to use this version during building, and it is not recommended to try and change this for use within the IceTray system." 
According to Don that's wrong, newer version can be used with no problem, and he asked me to submit this as a ticket.

Migrated from https://code.icecube.wisc.edu/ticket/1362
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:09", 
    "description": "On\nhttp://software.icecube.wisc.edu/offline_trunk/metaproject/quickstart.html\nit says that \"ROOT (http://root.cern.ch/drupal/) is supplied with the Port-based tool system. Currently we provide ROOT v5.28.00d. The I3Build system will automatically choose to use this version during building, and it is not recommended to try and change this for use within the IceTray system.\" \nAccording to Don that's wrong, newer version can be used with no problem, and he asked me to submit this as a ticket.", 
    "reporter": "chraab", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335649133028", 
    "component": "combo core", 
    "summary": "Documentation states that ROOT 5.28 is the ports version", 
    "priority": "major", 
    "keywords": "docs,documentation,root,install", 
    "time": "2015-09-22T09:37:03", 
    "milestone": "", 
    "owner": "nega", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150922T09:37:32: chraab changed attachment from "" to "quickstart_docs.png"

**Comment by icecube-trac** [comment]
Trac update at 20151029T10:57:26: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20151209T16:42:04: 
* david.schultz changed type from "rumor / allegation" to "cleanup"
* david.schultz changed milestone from "" to "January 2016"
* david.schultz changed priority from "normal" to "major"

**Comment by icecube-trac** [comment]
Trac update at 20151209T21:29:41: 
* nega changed status from "assigned" to "closed"
* nega commented:

> In changeset 140282:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140282"
> update referenced root version
> 
> closes: #1354
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:09: 
* olivas commented:

> Milestone January 2016 deleted


* olivas changed milestone from "January 2016" to ""
